lib/jekyll-feed/generator.rb

```diff
@@ -48,6 +49,22 @@ def feed_path(collection: "posts", category: nil)
       collections.dig(collection, "path") || "#{prefix}.xml"
     end
 
+    # Determines the title of a given feed
+    #
+    # collection - the name of a collection, e.g., "posts"
+    # category - a category within that collection, e.g., "news"
+    #
+    # Will return the site title or name
+    # ...followed by collection title or capitalized name
+    # ...followed by capitalized category name
+    def feed_title(collection: "posts", category: nil)
+      words = []
+      words << (@site.config["title"] || @site.config["name"])
```

Comment:
```suggestion
      words << @site.config["title"] || @site.config["name"]
```

Reply:
That's what I had initially written, but due to operator precedence Ruby reads it as: "add title to words, or return name" and test `JekyllFeed parsing with site.name set uses site.name for the title` fails. If there's a more elegant syntax let me know.

Reply:
The more elegant way would be using a local variable. But that's just unnecessary allocation. So let's leave this as it is..
```ruby
title = @site.config["title"] || @site.config["name"]
words << title
```